openedx/features/partners/tests/test_login.py

```diff
@@ -0,0 +1,96 @@
+from django.urls import reverse
+from openedx.features.partners.tests.factories import PartnerUserFactory, PartnerFactory
+from rest_framework.test import APITestCase
+from rest_framework import status
+from student.tests.factories import UserFactory
+
+
+class ResetPasswordTestCases(APITestCase):
+
+    def setUp(self):
+        self.end_point = reverse('partner_login',args=['give2asia'])
+
+    def test_login_with_correct_credentials(self):
+        valid_data = {
+            'email': 'abc@test.com',
+            'password': 'Abc12345'
+        }
+        user = UserFactory(username='testuser', password='Abc12345',email='abc@test.com')
+        partner_1 = PartnerFactory(label="partner_1", main_logo='abc', small_logo='xyz', slug='give2asia')
```

Comment:
Lets update factory so that redundant data like `main_logo, small_logo` should not be required each time

Reply:
Done.